lib/Router.js

```diff
@@ -100,6 +100,16 @@ Router.prototype.route = function(patternConfig, schema, handlers) {
                 query[queryParam] = parseFloat(query[queryParam]);
                 break;
 
+            case 'boolean':
```

Comment:
there's also another casting that you can make in Router.prototype.handle ... i'm preparing another PR to put them all in one function